index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="desktop-style.css" />
+    <link rel="stylesheet" href="mobile-style.css" />
+    <title>Capstone</title>
+  </head>
+  <body>
+    <header class="black-menu hide">
+      <ul class="black-menu hide">
+        <li>
+          <a href="#"
+            ><img
+              class="hide link"
+              src="Images/facebook.png"
+              alt="facebook-logo"
+              height="20"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              class="hide link"
+              src="Images/twitter.png"
+              alt="twitter-logo"
+              height="20"
+          /></a>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </header>
+
+    
+    <div class="menu">
+      <a href="#" class="disapeer" id="hamburger-menu"
+        ><img
+          class="disapeer"
+          src="Images/menu-mobile.png"
+          alt="hamburger-menu"
+          width="60"
+      /></a>
+      <a class="link hide" href="index.html"
+        ><img
+          class="hide right"
+          src="Images/logo-villa.png"
+          alt="logo"
+          width="135"
+      /></a>
+      <ul class="menu-section">
+        <li>
+          <span class="material-icons close disapeer">close</span>
+        </li>
+        <li><a class="link" href="about.html">About</a></li>
```

Comment:
- [ ] OPTIONAL 
It is impossible to get back to the _homepage_ when a visitor is currently on the _about us_ page. Kindly ensure to add a home element to the menu item which should link users to the homepage